health_check/urls.py

```diff
@@ -2,10 +2,11 @@
 from django.conf.urls import url
 
 import health_check
-from health_check.views import home
+from health_check.views import home, json_view
 
 health_check.autodiscover()
 
 urlpatterns = [
     url(r'^$', home, name='health_check_home'),
+    url(r'^json/?$', json_view, name='health_check_json_view')
```

Comment:
What do you think of not add a separate path but making the view header sensitive? You could request the response to be a json.
```
Accept: application/json
```